streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml

```diff
@@ -133,7 +133,7 @@
              limit 1
     </select>
 
-    <select id="getAffectedJobsByClusterId" resultType="java.lang.Integer" parameterType="java.lang.Long">
+    <select id="countJobsByClusterId" resultType="java.lang.Integer" parameterType="java.lang.Long">
         select
             count(1)
         from t_flink_app
```

Comment:
Do we need to filter the status of the application here? https://github.com/apache/incubator-streampark/pull/2809#discussion_r1242367682

Reply:
> Do we need to filter the status of the application here? [#2809 (comment)](https://github.com/apache/incubator-streampark/pull/2809#discussion_r1242367682)

Thanks for your feedback. I agree with you. I think We should filter the status of the application